### PR TITLE
Jia/fix: text align issue for our locations

### PR DESCRIPTION
**Pull request by jia-deriv**
Fix text align issue for our location section in who we are page
image

### CODE REVIEW

File: libs/blocks/src/lib/trade-types/index.tsx [hunk 1]
```diff
@@ -22,10 +28,14 @@ export const TradeTypeBlock = <T extends CardVariantType>({
...
-        <Heading.H2>{title}</Heading.H2>
+      <FluidContainer className="flex flex-col items-center justify-center">
+        <Heading.H2
+          className={qtJoin(description ? 'pb-gap-xl' : 'pb-gap-3xl')}
+        >
+          {title}
+        </Heading.H2>
         {description && (
-          <Text size="lg" className="text-typography-default">
+          <Text size="lg" className="pb-gap-3xl text-typography-default">
```
Comment (Nuzhy-Deriv) [inline libs/blocks/src/lib/trade-types/index.tsx:38]: can remove `text-typography-default` here as well.
if you want to have padding-bottom, please use `pb-genenral-[value]`

File: libs/components/src/lib/card/base/index.tsx [hunk 2]
```diff
@@ -92,7 +92,9 @@ export const BaseCard: React.FC<BaseCardProps> = ({
               'flex flex-col',
               textColorVariantClass[color],
               sizeVariantTextGap[size],
+              // 'gap-gap-lg',
```
Comment (Nuzhy-Deriv) [inline libs/components/src/lib/card/base/index.tsx:95]: can remove this comment if it's not needed.

Comment (jia-deriv) [reply libs/components/src/lib/card/base/index.tsx]: Done with removing the commented line

File: libs/blocks/src/lib/trade-types/index.tsx (hunk 1 above)
Comment (jia-deriv) [reply libs/blocks/src/lib/trade-types/index.tsx]: Sorry is a wrong commit. This should be in TradeTypeBlock fix [PR](https://github.com/deriv-com/deriv-com-v2/pull/190)

**Review by mohsen-deriv — APPROVED** [review]

**Pull request merged**